WalletWasabi/Helpers/Constants.cs

```diff
@@ -94,7 +94,7 @@ public static class Constants
 
 	public static readonly Money MaximumNumberOfBitcoinsMoney = Money.Coins(MaximumNumberOfBitcoins);
 
-	public static readonly Version ClientVersion = new(2, 3, 0, 0);
+	public static readonly Version ClientVersion = new(2, 3, 1);
```

Comment:
I would not remove the trailing zero, this might break things

Reply:
we never used to do this, I wouldn't change it now

Reply:
I checked and it doesn't

Reply:
And if it does, better to break things for this release which is a minor one

Reply:
We also had one release like that already
https://github.com/WalletWasabi/WalletWasabi/releases/tag/v2.0.8

Reply:
> We also had one release like that already https://github.com/WalletWasabi/WalletWasabi/releases/tag/v2.0.8

it still had the trailing zero in the code, 2.0.8 was just the title https://github.com/WalletWasabi/WalletWasabi/blob/7f46a8e95361e7789b8cf2c05b5c9ec41a73b469/WalletWasabi/Helpers/Constants.cs#L87

Reply:
to try with minor release is fine indeed.
I just remember another time where client notification was broken, because of something like this.
I don't remember exactly right now